zkvm-prover/Cargo.toml

```diff
@@ -5,38 +5,26 @@ edition = "2021"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
-[patch.crates-io]
-alloy-primitives = { git = "https://github.com/scroll-tech/alloy-core", branch = "v0.8.21" }
-ruint = { git = "https://github.com/scroll-tech/uint.git", branch = "v1.12.3" }
-tiny-keccak = { git = "https://github.com/scroll-tech/tiny-keccak", branch = "scroll-patch-v2.0.2-openvm-v1.0.0-rc.1" }
-
 [dependencies]
-anyhow = "1.0"
-log = "0.4"
-env_logger = "0.11.3"
-serde = { version = "1.0.198", features = ["derive"] }
-serde_json = "1.0.116"
+scroll-zkvm-types.workspace = true
+scroll-zkvm-prover-euclid.workspace = true
+sbv-primitives.workspace = true
```

Comment:
I guess we dont need sbv?

Reply:
Wait this is zkvm-prover Cargo toml, not top level.   It is not related to blockchain At all?

Reply:
Oh yes. For zkvm-prover we can consider using alloy-primitive instead of sbv-primitives

Reply:
Why need alloy primitive?

Reply:
ok